Stonith/docs/3nodes/scsi_stonith.md

```diff
@@ -0,0 +1,683 @@
+### Setup on 3 VM's
+
+1. Did pcs cluster setup on below 3 nodes by following the documents: 
```

Comment:
Can you separate scsi and ipmi
and add in https://github.com/Seagate/cortx-experiments/tree/main/Stonith/docs
depend on scsi and ipmi

Reply:
Yeahh agree, Separated out them in a different files.

Reply:
can we move in
Stonith/docs/scsi/3node_scsi_stonith.md
Stonith/docs/ipmi/3node_ipmi_stonith.md
scsi and ipmi are already present

Reply:
Done